### ISSUE TITLE
Problems with Style/IfUnlessModifier in case Layout/Tab is disabled

### ISSUE THREAD

**Issue by qnet-herwin** [rubocop-hq/rubocop#6124]
Simple PoC:

rubocop config:
```
Layout/Tab:
  Enabled: false
```

ruby file:
```ruby
if a
	puts 'Inline hint'  # This line doesn't even have to start with a tab, spaces work too to reproduce the bug
end
```

## Actual behavior

This results in an error:
```
An error occurred while Style/IfUnlessModifier cop was inspecting /home/herwin/tmp/rubocop_layout_tab_style_ifunlessmodifier/file.rb:1:0.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
```

## RuboCop version

```
$ bundle exec rubocop -V
0.58.2 (using Parser 2.5.1.2, running on ruby 2.3.3 i386-linux-gnu)
```

But seen this behaviour with 0.58.0 and 0.58.1 as well.

**Comment by qnet-herwin** [comment]
(For my own references: Q157145)

**Comment by AlexWayfer** [comment]
The same. Probably related to #5785.

```
An error occurred while Style/IfUnlessModifier cop was inspecting /home/alex/Projects/ruby/flame-menu/lib/flame/menu.rb:36:3.
nil can't be coerced into Integer
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/mixin/statement_modifier.rb:42:in `*'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/mixin/statement_modifier.rb:42:in `length_in_modifier_form'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/mixin/statement_modifier.rb:35:in `modifier_fits_on_single_line?'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/mixin/statement_modifier.rb:14:in `single_line_as_modifier?'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/style/if_unless_modifier.rb:51:in `eligible_node?'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/style/if_unless_modifier.rb:35:in `on_if'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:57:in `block (2 levels) in trigger_responding_cops'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:105:in `with_cop_error_handling'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:56:in `block in trigger_responding_cops'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:55:in `each'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:55:in `trigger_responding_cops'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:130:in `on_if'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `block in on_begin'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `each'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `on_begin'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:95:in `on_def'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `block in on_begin'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `each'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `on_begin'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:141:in `on_while'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `block in on_begin'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `each'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:49:in `on_begin'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/ast/traversal.rb:12:in `walk'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/commissioner.rb:45:in `investigate'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/team.rb:114:in `investigate'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/team.rb:102:in `offenses'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cop/team.rb:44:in `inspect_file'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:265:in `inspect_file'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:212:in `block in do_inspection_loop'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:244:in `block in iterate_until_no_changes'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:237:in `loop'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:237:in `iterate_until_no_changes'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:208:in `do_inspection_loop'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:111:in `block in file_offenses'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:129:in `file_offense_cache'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:109:in `file_offenses'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:100:in `process_file'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:78:in `block in each_inspected_file'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:75:in `each'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:75:in `reduce'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:75:in `each_inspected_file'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:67:in `inspect_files'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/runner.rb:39:in `run'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cli.rb:157:in `execute_runner'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cli.rb:85:in `execute_runners'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/lib/rubocop/cli.rb:41:in `run'
/home/alex/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rubocop-0.58.2/exe/rubocop:13:in `block in <top (required)>'
```

**Comment by AlexWayfer** [comment]
I'm fixing. Sorry for this error.

Work-around (and expected behavior): add `Layout/Tab: IndentationWidth: N`, where `N` is tab-width.
